releases/geneva/compose-files/docker-compose-geneva.yml

```diff
@@ -0,0 +1,585 @@
+# /*******************************************************************************
+#  * Copyright 2018 Dell Inc.
+#  *
+#  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+#  * in compliance with the License. You may obtain a copy of the License at
+#  *
+#  * http://www.apache.org/licenses/LICENSE-2.0
+#  *
+#  * Unless required by applicable law or agreed to in writing, software distributed under the License
+#  * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  * or implied. See the License for the specific language governing permissions and limitations under
+#  * the License.
+#  *
+#  * @author: Jim White, Dell
+#  * EdgeX Foundry, Fuji, version 1.1.0
+#  * added: Jun 30, 2019
+#  *******************************************************************************/
```

Comment:
Can we updated this copyright section?

This applies to the other `docker-compose` files in this PR as well.

Reply:
Done